lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -41,7 +46,7 @@ feature_nv_overlay:
 mapped_vlan:
   config_get: 'show running vlan'
   config_get_token: ['/^vlan <vlan>$/', '/^vn-segment (\d+)$/']
-  config_set: ['vlan <vlan>', '<state> vn-segment <vni>']
+  config_set: ['vlan <vlan>', '<state> vn-segment <vni> ; end']
```

Comment:
Yes we need it. Without the ; end, the result[2][body] is empty. It doesn't contain any message.